apps/element-storybook/src/508-compliance.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/addon-docs';
+import storybook_accessibility from '../../../static/storybook_accessibility.png';
+
+<Meta title="508 Compliance" />
```

Comment:
Can we swap to just say "Accessibility", more clear than saying "508 Compliance" everywhere. Maybe just in one spot specify that the level of accessibility we're striving for is what's required by section 508, possibly specifying which WCAG level Section 508 currently specifies.

Reply:
done